### PR TITLE
Details tabs clusters partners #211

### LINKED ISSUE #211: Add activities, partners (some screens) tabs to cluster objective, cluster activity and partner project detail screens in cluster reporting
Mockups: PRP cluster reporting

Nav: Everything under response parameters, my planned action.

**Technical notes**
#201 should have put all necessary API's needed in place by now.

### PR DESCRIPTION
**Pull request by ErinOC**
Add activities, partners (some screens) tabs to cluster objective, cluster activity and partner project detail screens in cluster reporting 

This resolves #211.

Changes in this PR:
+ Added basic components for each tab (content of the indicators tabs will be handled in a separate ticket).
Added content for:
+ Response Parameters —> Clusters —> Cluster Objective —> Activities
+ Response Parameters —> Clusters —> Cluster Activity —> Contributing Partners
+ Response Parameters —> Partners—> Projects —> Activities
+ My Planned Action—> Projects—> Activities
Misc:
+ Moved clusterActivities to separate Actions and Reducer files
+ Added a partner dropdown filter (API returning an empty list currently)
+ Increased the submenu left padding (it was misaligned after the addition of icons)

Note: I filed a ticket for some known issues in #211: https://github.com/unicef/etools-partner-reporting-portal/issues/245

##### Progress checker
- [ ] Polymer

- [x] Polymer test cases

### CODE REVIEW

**Comment by ErinOC** [comment]
Note that I had to merge #201 (backend API work) into this branch to proceed. #201 hasn't been approved yet. @n1207n

File: polymer/src/elements/cluster-reporting/response-parameters/clusters/objectives/activities.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+        this.dispatch(App.Actions.clusterActivities.fetchClusterActivitiesList(thunk))
+          .catch(function (err) { // jshint ignore:line
+              // TODO: error handling.
+          });
+      },
+
+      _computeActivitiesUrl: function () {
+        //Make sure the queryParams are updated before the thunk is created:
+        this.set('queryParams.cluster_objective_id', this.objectiveId);
```
Comment (rkrupinski) [inline polymer/src/elements/cluster-reporting/response-parameters/clusters/objectives/activities.html]: Please, don't set stuff while computing props, use observers instead.

File: polymer/src/elements/cluster-reporting/planned-action/projects/activities.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        },
+      },
+
+      behaviors: [
+        App.Behaviors.UtilsBehavior,
+        App.Behaviors.ReduxBehavior,
+      ],
+
+      observers: [
+        '_activitiesByPartnerProjectIdAjax(queryParams)',
```
Comment (rkrupinski) [inline polymer/src/elements/cluster-reporting/planned-action/projects/activities.html]: No need to use complex observers for top-level props.

File: polymer/src/elements/cluster-reporting/planned-action/projects/activities.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      ],
+
+      _computeUrl: function (projectId) {
+        //Make sure the queryParams are updated before the thunk is created:
+        this.set('queryParams.page', 1);
+        this.set('queryParams.project', projectId);
```
Comment (rkrupinski) [inline polymer/src/elements/cluster-reporting/planned-action/projects/activities.html]: Please, avoid setting stuff while computing props, use observers instead.

**Pull request merged**